lib/widget/like_button.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+
+class LikeButton extends StatefulWidget {
+  int like;
+
+  LikeButton(this.like, {super.key});
```

Comment:
trailing comma

Reply:
Also you can add all the parameter inside the {}

Reply:
done